services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -642,6 +643,11 @@ protected void initAttesterSlashingPool() {
                 .reversed());
     blockImporter.subscribeToVerifiedBlockAttesterSlashings(attesterSlashingPool::removeAll);
     attesterSlashingPool.subscribeOperationAdded(forkChoice::onAttesterSlashing);
+    final ValidatorTimingChannel validatorTimingChannel =
+        eventChannels.getPublisher(ValidatorTimingChannel.class);
+    attesterSlashingPool.subscribeOperationAdded(
+        (operation, validationStatus, fromNetwork) ->
+            validatorTimingChannel.onAttesterSlashing(operation));
```

Comment:
we only want this if we're using this feature, otherwise we're permanently listening locally to the slow version of the message?

Reply:
Good catch. Fixed here [7918](https://github.com/Consensys/teku/pull/7918/files#diff-a0738cefef0ee011967c18c1fe03e9c169ab10cfc7a1e7531f31cd71505b7a39)